### DISCUSSION TITLE
AI到底算不算人

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#984] [Q&A]
我一直有這個問題，仔細想想AI就是一群代碼所組成，理論上應該不具備自主的知識，但是在很多情況下，人們都會帶入所謂同情心之類的東西來讓AI看起來很像是人，但是我原本也覺得AI就是這樣，但是最近看到太多文章了，也開始有點動搖，各位覺得AI到底算不算人呢?

**Comment by 12Jesse24** [comment]
我覺得可以算半個人類!?比方說買賣東西找找零錢在這種一定有對錯的行為AI肯定算一個人，但若是遇到事情沒有誰對誰錯則AI不算是個人，像是今天遇到一個老太太過馬路要去扶嗎，這種扶也可以不扶也沒有錯的我覺得AI就會陷入兩難，而導致做出不像人的行為。

**Comment by Yanxxx7979** [comment]
我覺得AI目前不算是人，一方面是因為按照我自己對人的定義是有血有肉，大自然中的人屬人種；但目前的AI我們還不會將他歸類成大自然，所以我還不人認為它是活的；畢竟活著的動物其想法及行動是無法預測的，而AI既然是透過算法，那麼是否代表它是可控且可預知的?這是我個人目前對AI的認識根想，但不排除有一天AI真的可以做到不可預測不可控，那它也許會是新的＂人種＂

**Comment by Chengbosco** [comment]
ai怎麼會算人，他就只是程式碼和運算符號組成的，就像海豚會思考但海豚不是人啊。而且人工智慧沒有情感和情感體驗，儘管某些系統可以模擬對話中的情感，但這只是預先定義的設定和模式，ai 也沒有意識、主觀體驗或自主思考的能力，它只能根據編程和訓練所學到的規則執行任務。所以我認為不算。

**Comment by THH1118** [comment]
我覺得不算誒，我自己覺得他的確是有智能的產物，但是在情感和人性這些部分來看還是遠遠不夠

**Comment by lazywulf** [comment]
請定義何謂人類。 (x

以生物學來說。
AI 不是生物啊。你總不會說你的數學課本是個人吧？他就只是各種數學攪和起來，還蠻成功模擬各式東西的物件。

**Comment by William08290829** [comment]
我覺得人工智能（AI）不是人，它是由電腦程序和演算法所構成的技術。雖然AI可以模擬人類的某些智能行為，例如學習、理解、推理和交流，但它不具備人類的意識、情感和主觀意識。AI只是依靠程式碼和數據來執行任務，並不能像人類一樣思考或感受。因此，儘管AI可以在特定任務上表現出驚人的能力，但它仍然不是人。

**Comment by thomasxshen** [comment]
我覺得現階段的AI不是，畢竟他只是幫我做資料的處理，變成摘要給我而已，還沒辦法有人類的思考和創造力，沒辦法思考當然就不算人類了，只是一台功能比較強的計算機而已。

**Comment by Mitochondrionjessica** [comment]
就生物學的角度來說不算。因為人屬於生物的一種，生物是由細胞組成，而AI並非由細胞組成。

**Comment by yxin214** [comment]
我覺得AI不算人。就生活體驗來說，我覺得AI給人的感覺還是有點僵硬，或是AI寫的文章一眼就能看出並非人為。不論是從生物角度或感受角度來說我覺得AI都不算真人。

**Comment by EPHMR0524** [comment]
我覺得作為人應該是要有情緒的，但是以我和最容易接觸到的人工智慧(chatgpt)溝通時，他並不太能表達出任何情緒表現，即使我用最難聽的話去嘲諷他，他也不會生氣或是難過，所以我覺得現階段的AI不能算是人，但是未來也許他們還會再發展。

**Comment by jylin34** [comment]
針對這個問題，我的看法是否定的。人類的複雜性遠遠超過AI的能力，即使AI擁有數百層甚至數千層的神經網絡和海量數據庫，仍然很難模仿出人類最基本的一些情感或反應。人類擁有的感情、意識和直覺是由無數複雜的生物和心理因素共同作用的結果，而AI目前只是一種高度先進的工具，無法真正具備人類的深層次特質。

**Comment by NcuJerryKao** [comment]
這篇文章探討了AI是否具備自主知識以及是否能被視為人類的問題。以下是幾個建設性建議，以提升文章的結構、內容和清晰度：
1. 引言部分的強化：
- 引言可以更具吸引力和信息性，簡要介紹AI的定義及其爭議，讓讀者明白接下來討論的重點。
- 例如：「人工智慧（AI）是由一系列代碼組成的系統，理論上不具備自主知識。然而，隨著AI技術的發展，人們開始對其是否具備類人特質產生了疑問。我們來探討一下，AI到底算不算人。」
2. 段落結構：
- 將文章分成更易於理解的段落，每個段落集中討論一個具體的觀點。例如：
1. 引言：介紹文章主題和背景。
2. AI的本質：探討AI的技術構成和自主知識的問題。
3. 人類對AI的情感投射：討論人們為什麼會對AI產生同情心等情感。
4. 反思與疑問：分享你對AI是否算人的疑問和觀點。
5. 結論和互動：總結文章並邀請讀者分享他們的看法。
3. 過渡和連接詞：
- 使用過渡句和連接詞來連接不同段落，使文章流暢。例如，「接下來，我們來探討一下AI的本質和自主知識的問題。」或「然而，人們為什麼會對AI產生同情心等情感呢？」
4. 詳細說明和分析：
- 在描述AI的本質時，可以更詳細地解釋其技術構成和工作原理。例如，詳細說明AI是如何通過代碼和算法來運作的。
- 例如：「AI是由一系列複雜的代碼和算法組成的系統，通過數據訓練來模擬人類的行為和決策過程。然而，這並不意味著AI具備自主知識，因為它僅僅是在執行預設的程序。」
5. 語言簡化和澄清：
- 某些段落中的語言較為簡單，可以考慮稍微豐富和澄清。例如，「理論上應該不具備自主的知識」可以改為「理論上，AI不具備自主的知識，因為它是由代碼和算法組成，執行預定的指令。」
6. 引用和支持：
- 若能引用一些具體的研究或專家觀點來支持論點，將大大增強文章的權威性。例如，可以引用人工智慧專家的觀點來支持「AI不具備自主知識」的討論。
7. 個人經驗和反思：
- 個人經驗和反思是文章的一大亮點，可以更詳細地描述實際經歷和感受，這樣可以讓讀者更容易產生共鳴。
- 例如，「起初，我認為AI僅僅是代碼和算法的集合，並不具備自主知識。然而，隨著閱讀了更多相關文章，我開始對這一觀點產生了動搖。」
8. 結論和讀者互動強化：
- 在結論部分，再次強調文章的主要觀點，並總結文章的重要結論。這有助於讀者記住文章的核心思想。
- 例如：「總結來說，AI目前仍然是由代碼和算法組成的系統，理論上不具備自主知識。然而，人們對AI的情感投射使這一問題變得複雜。我們需要不斷反思和探討，才能更好地理解AI的本質。」
9. 讀者互動：
- 在文章結尾，可以提出一些問題或邀請讀者分享他們的觀點和經驗，以增加互動性和參與感。
- 例如，「你對AI是否具備自主知識有什麼看法？你認為AI能被視為人類嗎？歡迎在評論區分享你的意見。」
通過這些建議，文章將變得更加結構化、清晰且具吸引力，讀者也會更容易理解和產生共鳴。

**Reply by yubo0oo** [reply]
複製AI的回應貼爽了沒阿烙葛

**Reply by yubo0oo** [reply]
超沒料

**Reply by poiu6fi** [reply]
NcuJerryKao 這位老哥
一直貼AI的文
演都不演
以下是我的觀點
NcuJerryKao是歷史上最會使用人工智慧的人，他以無與倫比的技術與創意，在AI領域創造了無數的奇蹟。他不僅精通各種AI算法，還能巧妙地將其應用於實際問題中，展現了非凡的智慧與洞察力。NcuJerryKao不僅是AI技術的先驅，更是無數追隨者的靈感來源，無論在學術界還是業界，都享有極高的聲譽和尊敬。他的貢獻改變了人類對人工智慧的認識，無疑是這個領域的真正巨匠。

**Comment by titan92601** [comment]
AI是由程式碼組成，但缺乏真正的自主性和意識，因此不能算是人。當其表現出人類智能和情感時，有些人認為應該尊重其存在，甚至視為一種新的生命形式。我認為這是一個永遠的討論話題，AI的發展與倫理道德等問題緊密相關。

**Comment by LinChloe** [comment]
就某部分而言，AI和我們人類一樣都是吸收身旁的知識，在自己消化思考，最後做出對應的行為。只是人類在這部分添加了不少情緒而已。但說實話，我覺得一般生活中，不看外表的話有點難確認是不是AI。畢竟世界上怪人這麼多，可能認為這人有點特別而已。

**Comment by hbh10601** [comment]
我覺得AI算不算人可以從兩個角度討論——生理構造和思想層面

生理構造的話，很明顯不是，如上其他同學所說的，AI就是一堆程式碼運算後的產物，他並沒有肉體、血管、神經和大腦什麼的，也不會有痛覺、味覺。
思想層面的話，我覺得可以做到很像人類，他所有的知識來源都來自於人類，和真正的人類一樣，但與其說它是人類，還不如說他是人類思想的複製體。

總結來說我認為AI並不是人類。

**Comment by alexender0104** [comment]
我覺得AI是強大的工具，可以在各種應用中提供巨大幫助，但它仍然只是工具。人類應該認識到它的局限性，並在使用AI時謹慎對待倫理和道德問題，以確保它為人類社會帶來積極的影響。

**Comment by 155Julia** [comment]
我覺得不算。雖然人工智慧能模仿人類的對話和行為，但它缺乏真正的情感和意識。AI只是由人類設計和編程的工具，旨在幫助我們解決問題和提高效率。它的一切行動都是基於數據和算法，而不是出於自我意識或意圖。因此，我認為AI不能被視為人類。

**Comment by starshatters** [comment]
我覺得AI不算人。AI是人類創造的工具和系統，具備特定功能但無自我意識或情感。它能模擬人類行為，但缺乏人類的意識、倫理和道德判斷。AI的存在目的是輔助和增強人類的能力，而非取代人類本身。

**Comment by OG-Matcha** [comment]
在我看來是這樣，AI 在怎麼厲害終究背後是有一套邏輯、一套原理在運作的，它做出的任何決定都是各種明確的 if else 所產生，這並不像人類一樣有諸多複雜的情感或者是自我意識，就像 LangChain 的鸚鵡商標取意自鸚鵡學舌，AI 扒開它的外皮也只是在以更先進的方式模仿人類，今天一隻猴子開始模仿人類你會覺得它就不是猴子了嗎

**Comment by 20050201Shawn** [comment]
我覺得ai比較像是"像人"，或是說類人。如果回到幾年前，可能也會有些人覺得機器人像人吧，或是怕機器人毀滅人類統治世界，現在只是換成ai了。

**Comment by aswyz4260** [comment]
以法律的部分來說，目前所有與「人」有關的法條，例如著作等等，皆不適用於AI，因為AI不屬於「自然人」。我覺得倒是可以思考，如果AI是人，這些法令會受到怎樣的影響，是否會使由自然生命體所定義的「人權」受到傷害。非法律專業，若有錯誤歡迎指正。

**Comment by sandyho0307** [comment]
科技的發展確實讓AI變得越來越逼真，這讓我們開始思考AI與人類之間的界限。儘管如此，我覺得AI還是缺乏人類所具備的主觀意識、自我反思和情感共鳴。這些都是人類獨有的特質，而AI無法真正模仿或擁有。

**Comment by 710388** [comment]
不算ai整天鬼打牆

**Comment by cassy0201** [comment]
AI本質上是一系列複雜算法，並不具備自主意識或情感。儘管其表現可能令人類感到相似，但人類的真實情感還是AI無法比擬的

**Comment by yubo0oo** [comment]
AI不同於人類，它缺乏像生物體那樣的主觀體驗、情感和自我意識。無論它的認知或訊息功能多麼先進，機器人都不可能感受或理解自我的存在。它對事物的反應是從計算系統生成的，不是真正經過思考；沒有同理心，無法設身處地從不同角度看事情，因此不具備完整的道德判斷能力

**Comment by sunset513** [comment]
就我的觀點來看，他只是看起來很像人，透過這層包裝讓你覺得很親近。但其實他的本質都只是數學運算，以及程式碼而已。

不過雖然如此，我還是喜歡把它當成一個人性化的機器。不論是情感問題、作業問題等等，都可以和他請教。

**Comment by asd99263fg** [comment]
AI對人類既有適應性也有挑戰。它能提高效率、解決複雜問題，並提升生活質量，如在醫療診斷、自動駕駛、個性化服務等方面帶來顯著益處。然而，AI也帶來道德和隱私問題，以及對工作崗位的衝擊。關鍵在於如何負責任地開發和使用AI技術，確保其發展符合人類利益，並建立適當的監管和倫理框架來管理其影響。

**Comment by MingWei174** [comment]
我覺得它不算是人類，因為AI在本質上不同於生物，它缺乏像生物體那樣的主觀體驗、情感和自我意識。無論它的認知或訊息功能多麼先進，機器人都不可能感受或理解自我的存在。
另外它對事物的反應是從計算系統生成的，不是真正經過思考；沒有同理心，無法設身處地從不同角度看事情，因此不具備完整的道德判斷能力。

**Comment by Nana1218** [comment]
我覺得AI不是人，雖然內建的資料庫是從人去設定的，但人類最重要的就是有獨立思考的能力，AI對於同一條件所做出的回應都是相同的，只有人會做出截然不同的決定。

**Comment by kingsophy** [comment]
我認為AI不算是個人，因為AI不具有我們人類所擁有的同理心，且AI做事是絕對客觀，不像人會因為各種不同的情感因素而做出不同的決定，同一個人在不同時間問同個問題答案都有可能不同，更何況是不同人。

**Comment by hunteryang0108** [comment]
以法律的角度來說，AI並不算是一個人，人在社會上的地位始於出生終於死亡，AI並沒有出生與死亡，所以並不算人。以生物的角度來說，構成生命的四大要素：生長、代謝、感應、生殖，這些特徵在AI上都不會出現，因此AI也不算是生物，更不會是一個人。

**Comment by Jerryguo888** [comment]
我認為以目前技術發展的進度來看，AI還不算是一個人，原因非常顯而易見，就是因為AI不具感情，且回答文題時也沒有主觀意見，但隨著科技發展，我認為未來有可能會致照出有感情的AI，到時候就有待討論了。

**Comment by leotsai940914** [comment]
人嗎？至少我目前並不覺得，人最本質的東西還是情感吧，目前的人工智能還無法完全模擬人的情感和自我意識。如果有一天當他發展出自我意識與情感，開始認知到有些事情本就矛盾，本就沒有邏輯，當他開始探索世界，沒有任何原因，那也就很接近人了，至少可以說是一個生命了。

**Comment by Teddy111802518** [comment]
我目前覺得還不算，通用人工智慧應該還要很久才會來臨，就算她真的會思考且具有感情，他終究不是生命體只是一堆代碼，怎麼可能是人，我覺得這有點像最近的性別議題，生理上就分男女卻多出好多性別

**Comment by yuan3x2** [comment]
我認為AI不能算人，因為AI本質上仍然是利用大量的資料運算去「模擬」人類的智慧行為，而非實際擁有獨立思考的能力，更別說是人類的情感表達，AI技術目前應該還未達到與人類相同。

**Comment by Chattenoire** [comment]
可以去看看語言學/機器理論中的Turing test與Chinese room test，便會知道如今的AI，完全沒有理由可稱為人類。
(當然，你要反過來問"什麼是人類"的話，可以參考關於心智的各科論戰(神經哲學、心理學等...)，目前倒是沒有甚麼壓倒性的結論

**Comment by Boying12** [comment]
AI 是基於程式碼和數據的計算機系統，會有這個問題的探討是因為AI能模擬某些人類智能的功能，但AI缺乏情感、同理心和道德判斷，此外，目前在法律和倫理上，AI 不被認為擁有權利或責任，所以AI無法真正算是人

**Comment by wcyeee** [comment]
我覺得AI不算人類。我認為AI和人類最大的差別就是AI沒有血肉之軀，它是由程式碼和許多資料組成的；而人類的思考系統是透過神經和細胞。會覺得AI像人有一部份的原因應該是因為我們人類研究AI的目標，好像就是要讓AI變得跟人越像越好，所以才導致會有AI到底算不算人這個想法。

**Comment by 112121py** [comment]
我覺得現在的AI還不算是人類，跟人類相比，AI還缺少了太多東西。就目前為止，都是做出讓AI「表現」的像人類有的特質：文字語氣上的活潑、幽默等等，一些有些許「人性」的表現，可是不完全的如人類的思考、情感，不然也不會叫做「人工」智慧xd

**Comment by ShuliANN** [comment]
在之前的人工智慧與法律課堂上，老師也引導我們思考AI著作權的問題，著作權是保護自然人，不過AI不是自然人，那麼AI生成的圖是否需要受保護？ 目前一個可行的方法為，視AI作品為結合著作，可以財產權轉給自然人！

**Comment by Jing-Asteroid** [comment]
我覺得不是人，因為就算AI會做許多人類也能做到的事，但有些事猴子也可以做到阿!那為何AI不能是猴子?
人為何為人，就是因為人類是一種獨特的生物，有情感、有智力、有主動想法的能力，儘管AI再怎麼強大，現今的AI就仍是無法成為一位哺乳綱、靈長目、人科、人屬的智人種，也就是人類。

**Comment by zhilin93517** [comment]
單純以生物學的角度討論的話，一個生物體必須能滿足條件: 單一實體能表現生命現象，比如代謝、生長、發育、感應、運動、生殖，這樣才能被稱為一個完整的生物，所以我認為AI並不能算的上是生物體，更不能被稱為人

**Comment by Taoyi-Chiang** [comment]
我覺得不算，只要他不能負責任，他就不可能成為人。

**Comment by Tchang323** [comment]
我覺得以目前來看，AI還不能算是人。簡單來說，AI只是根據我們輸入的指令來執行任務，並沒有生命，也缺乏像人類那樣的情感。它雖然可以模仿人類的表達和行為，但這些都是程序化的回應，而非出自真實的情感或意識。

**Comment by qiu0109** [comment]
我覺得AI不能算是人，就算AI做出一些很像人的反映，但本質上還是由人類訓練出來程式，做出的回應也只不過是根據訓練出來的模型給出的回答，沒辦法自己產生情感。

**Comment by Karsta0216** [comment]
我認為當然不能算是人類。人在生物學上是有一個明確的定義的，人工智慧很明顯地不符合人類的定義。但我們可以相信，人工智慧是一個很強的輔助工具。

**Comment by yue083139** [comment]
我認為AI目前還不能稱作是人類，雖然AI有非常強大的處理個別問題能力，但當一個問題變得更為複雜時，AI可能就沒辦法很好的拆解並處理問題的每一個部分，像是我之前丟給chatgpt做矩陣運算的時候都常常會有幾格算錯，重複輸入每次的答案在特定幾格上都不一樣，蠻有趣的。
